.github/workflows/presubmit-build.yaml

```diff
@@ -34,7 +34,17 @@ jobs:
         id: augmented-inputs
         env:
           EXTRA_INPUT_APKO_IMAGE: ghcr.io/wolfi-dev/apko:latest@sha256:d3585d22de39f9029df0f51c2d63e21698d9c82d52b892cf9d8c4f6b7c5eecaf
+          EXTRA_INPUT_APKO_REPOSITORY_APPEND: https://packages.wolfi.dev/os
+          EXTRA_INPUT_APKO_KEYRING_APPEND: https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+          EXTRA_INPUT_APKO_PACKAGE_APPEND: wolfi-baselayout
```

Comment:
I think the next step with this is to populate this directly in the `matrix` via `monopod` reading `globals.yaml` and `image.yaml`, so we can stop blending/specifying "options" and just rely on `--packages-append` / `extra_packages`.

cc @imjasonh @jdolitsky